axiom/util_number_converter.c

```diff
@@ -46,6 +47,22 @@ void nr_itoa(char* buf, size_t len, int x) {
   nr_strlcpy(buf, p, len);
 }
 
+#define PRIORITY_MAXLEN 6
+char* nr_priority_double_to_str(double value) {
+  char* buf = NULL;
+
+  buf = nr_formatf("%.6f", value);
```

Comment:
This format should use `PRIORITY_MAXLEN` instead of a hard coded constant.

Reply:
Should accept `nr_sampling_priority_t`.

Reply:
fixed here: 880b18e98ed259ea876d70bc91e43a018c0b949f